pkgs/top-level/all-packages.nix

```diff
@@ -6409,7 +6409,7 @@ in
 
   termplay = callPackage ../tools/misc/termplay { };
 
-  testdisk-photorec = callPackage ../tools/system/testdisk-photorec { };
+  testdisk-photorec = throw "This package was a duplicate, please use testdisk or testdisk-qt instead";
```

Comment:
Nitpick: perhaps use the usual `pkgs/top-level/aliases.nix`?

Reply:
@vcunat: Thanks for the pointer, I was actually not aware of this additional mechanism. Done.